ui/raidboss/data/06-ew/trial/barbariccia-ex.txt

```diff
@@ -1,7 +1,315 @@
+291.0 "--sync--" sync / 1[56]:[^:]*:Barbariccia:7597:/
+293.3 "Knuckle Drum" duration 7.5 #sync / 1[56]:[^:]*:Barbariccia:7598:/
+300.8 "--sync--" sync / 1[56]:[^:]*:Barbariccia:7599:/
+302.9 "--sync--" sync / 1[56]:[^:]*:Barbariccia:7582:/
+
+# Transition 2
+304.2 "Iron Out" sync / 1[56]:[^:]*:Barbariccia:75B5:/
+309.4 "Voidstrom" duration 2 #sync / 1[56]:[^:]*:Barbariccia:7577:/
+312.4 "Iron Out" sync / 1[56]:[^:]*:Barbariccia:7455:/
+318.5 "Raging Storm" sync / 1[56]:[^:]*:Barbariccia:7572:/
+
+# Playtation 1
+325.6 "Entanglement" sync / 1[56]:[^:]*:Barbariccia:75AD:/
+326.4 "Fetters" sync / 1[56]:[^:]*:Barbariccia:75B0:/
+327.9 "--sync--" sync / 1[56]:[^:]*:Barbariccia:75AF:/
+335.1 "Secret Breeze" sync / 1[56]:[^:]*:Barbariccia:7580:/
+336.1 "Secret Breeze 1" #sync / 1[56]:[^:]*:Barbariccia:7415:/
+338.2 "Secret Breeze 2" #sync / 1[56]:[^:]*:Barbariccia:7581:/
+
+# Pattern 3a, linked with pattern 4a through 757A+7575 and 757A+757B
+347.4 "Savage Barbery" sync / 1[56]:[^:]*:Barbariccia:(7573|7464|7465|7466|7489|748B):/
+348.4 "Savage Barbery" sync / 1[56]:[^:]*:Barbariccia:(7574|757A):/
+350.5 "Savage Barbery" sync / 1[56]:[^:]*:Barbariccia:(7575|757B):/
+
+351.5 "Brush with Death" sync / 1[56]:[^:]*:Barbariccia:75A4:/
+
+# Pattern 3b, linked with Pattern 4b through 757C+757D and 757E+757F and also independently by 75A6 and 75A7
+360.6 "Hair Raid" sync / 1[56]:[^:]*:Barbariccia:(757C|757E):/
+362.6 "Hair Raid" sync / 1[56]:[^:]*:Barbariccia:(757D|757F):/
+364.9 "Deadly Twist/Hair Spray" sync / 1[56]:[^:]*:Barbariccia:(75A7/75A6):/
+
+373.8 "Void Aero IV" sync / 1[56]:[^:]*:Barbariccia:7570:/
+380.9 "Void Aero III" sync / 1[56]:[^:]*:Barbariccia:7571:/
+388.1 "Raging Storm" sync / 1[56]:[^:]*:Barbariccia:7572:/
+
+# Playstation 2
+395.2 "Entanglement" sync / 1[56]:[^:]*:Barbariccia:75AD:/
+396.0 "Fetters" sync / 1[56]:[^:]*:Barbariccia:75B0:/
+396.0 "Fetters" sync / 1[56]:[^:]*:Barbariccia:75B0:/
+397.5 "--sync--" sync / 1[56]:[^:]*:Barbariccia:75AF:/
+400.4 "Brush with Death" sync / 1[56]:[^:]*:Barbariccia:75A3:/
+409.2 "Upbraid" sync / 1[56]:[^:]*:Barbariccia:75A8:/
+
+# Pattern 4a
+416.7 "Savage Barbery" sync / 1[56]:[^:]*:Barbariccia:(7573|7464|7465|7466|7489|748B):/
+417.7 "Savage Barbery" sync / 1[56]:[^:]*:Barbariccia:(757A|7574):/
+419.8 "Savage Barbery" sync / 1[56]:[^:]*:Barbariccia:(757B|7575):/
+
+# Pattern 4b
+426.8 "Hair Raid" sync / 1[56]:[^:]*:Barbariccia:(757C|757E):/
+428.8 "Hair Raid" sync / 1[56]:[^:]*:Barbariccia:(757D|757F):/
+430.7 "Hair Spray/Deadly Twist" sync / 1[56]:[^:]*:Barbariccia:(75A6|75A7):/
+
+439.9 "Void Aero IV" sync / 1[56]:[^:]*:Barbariccia:7570:/
+447.1 "Raging Storm" sync / 1[56]:[^:]*:Barbariccia:7572:/
+
+# Transition 3 - Second Curling Iron
+455.4 "Curling Iron" sync / 1[56]:[^:]*:Barbariccia:75B2:/
+460.7 "Voidstrom" duration 2 #sync / 1[56]:[^:]*:Barbariccia:7577:/
+463.7 "Curling Iron" sync / 1[56]:[^:]*:Barbariccia:75B3:/
+
+468.8 "Brutal Rush 1" sync / 1[56]:[^:]*:Barbariccia:7583:/
```

Comment:
Did you run this through test_timeline? I'm surprised this worked with all of the repeated ids here next to each other with overlapping syncs.  I think for something like these, maybe it'd make sense to just have a duration? I don't think having all of these listed is as helpful as something like sanctity meteor counts.

Reply:
I never ran it through test timeline, but I think it did work in game, or worked enough.
I was thinking it should be changed to duration, it's odd for two spells, but for the most part I think it would be more functional as a duration since one part is the hit and the other is the line.